src/Build/Resources/Strings.resx

```diff
@@ -1324,6 +1324,12 @@
     <value>Could not resolve SDK "{0}". Exactly one of the probing messages below indicates why we could not resolve the SDK. Investigate and resolve that message to correctly specify the SDK.
   {1}</value>
   </data>
+  <data name="SucceededToResolveSDK" xml:space="preserve">
+    <value>The SDK "{0}" was successfully resolved by the "{1}" resolver to location "{2}" and version "{3}".</value>
+  </data>
+  <data name="SDKResolverAttempt" xml:space="preserve">
+    <value>The "{0}" resolver attempted to resolve the SDK "{1}". {2}</value>
```

Comment:
Here it is not obvious what {2} is. Localization team might need to know. Please include loc hints like for example:
``` XML
    <comment>
      LOCALIZATION: "{2}" is new line separated errors and warnings or empty if no errors or warnings.
    </comment>
```